Samples/outlook-check-item-categories/manifest.xml

```diff
@@ -0,0 +1,144 @@
+              <!-- JavaScript file that contains the event handlers. This is used by Outlook on Windows. -->
+              <Override type="javascript" resid="JSRuntime.Url"/>
+            </Runtime>
+          </Runtimes>
+          <DesktopFormFactor>
+            <ExtensionPoint xsi:type="MessageComposeCommandSurface">
+              <OfficeTab id="TabDefault">
+                <Group id="CategoriesGroupMessageCompose">
+                  <Label resid="CategoriesGroupLabel"/>
+                  <Control xsi:type="Button" id="MessageComposeSelectButton">
+                    <Label resid="SelectButtonLabel"/>
+                    <Supertip>
+                      <Title resid="SelectButtonLabel"/>
+                      <Description resid="SelectButtonDescription"/>
+                    </Supertip>
+                    <Icon>
+                      <bt:Image size="16" resid="Icon.16x16" />
+                      <bt:Image size="32" resid="Icon.32x32" />
+                      <bt:Image size="80" resid="Icon.80x80" />
+                    </Icon>
+                    <Action xsi:type="ShowTaskpane">
+                      <SourceLocation resid="Taskpane.Url"/>
+                    </Action>
+                  </Control>
+                </Group>
+              </OfficeTab>
+            </ExtensionPoint>
+            <ExtensionPoint xsi:type="AppointmentOrganizerCommandSurface">
+              <OfficeTab id="TabDefault">
+                <Group id="CategoriesGroupAppointmentOrganizer">
+                  <Label resid="CategoriesGroupLabel"/>
+                  <Control xsi:type="Button" id="AppointmentOrganizerSelectButton">
+                    <Label resid="SelectButtonLabel"/>
+                    <Supertip>
+                      <Title resid="SelectButtonLabel"/>
+                      <Description resid="SelectButtonDescription"/>
+                    </Supertip>
+                    <Icon>
+                      <bt:Image size="16" resid="Icon.16x16" />
+                      <bt:Image size="32" resid="Icon.32x32" />
+                      <bt:Image size="80" resid="Icon.80x80" />
+                    </Icon>
+                    <Action xsi:type="ShowTaskpane">
+                      <SourceLocation resid="Taskpane.Url"/>
+                    </Action>
+                  </Control>
+                </Group>
+              </OfficeTab>
+            </ExtensionPoint>
+            <!-- Enables the add-in on specified events. -->
+            <ExtensionPoint xsi:type="LaunchEvent">
+              <LaunchEvents>
+                <LaunchEvent Type="OnNewMessageCompose" FunctionName="onItemComposeHandler"/>
+                <LaunchEvent Type="OnNewAppointmentOrganizer" FunctionName="onItemComposeHandler"/>
+                <LaunchEvent Type="OnMessageSend" FunctionName="onItemSendHandler" SendMode="SoftBlock"/>
+                <LaunchEvent Type="OnAppointmentSend" FunctionName="onItemSendHandler" SendMode="Block"/>
```

Comment:
An option is to have a unique function name for each event then internally point to the same handler. Then in future you can handle independently without needing to update the manifest. For example:
```suggestion
                <LaunchEvent Type="OnNewMessageCompose" FunctionName="onMessageComposeHandler"/>
                <LaunchEvent Type="OnNewAppointmentOrganizer" FunctionName="onAppointmentComposeHandler"/>
                <LaunchEvent Type="OnMessageSend" FunctionName="onMessageSendHandler" SendMode="SoftBlock"/>
                <LaunchEvent Type="OnAppointmentSend" FunctionName="onAppointmentSendHandler" SendMode="Block"/>
```